Swiftfin/Views/PagingLibraryView/PagingLibraryView.swift

```diff
@@ -260,18 +291,18 @@ struct PagingLibraryView<Element: Poster>: View {
             switch viewModel.state {
             case .content:
                 if viewModel.elements.isEmpty {
-                    L10n.noResults.text
+                    contentLetterBarView(content: L10n.noResults.text)
                 } else {
-                    contentView
+                    contentLetterBarView(content: contentView)
                 }
             case let .error(error):
-                errorView(with: error)
+                contentLetterBarView(content: errorView(with: error))
             case .initial, .refreshing:
-                DelayedProgressView()
+                contentLetterBarView(content: DelayedProgressView())
             }
         }
         .animation(.linear(duration: 0.1), value: viewModel.state)
-        .ignoresSafeArea()
+        // .ignoresSafeArea()
```

Comment:
This currently only impacts non-romantic languages (Korean, Japanese, etc.) where I change the Letter Picker from a static VStack to a Scroll. I think this may lead into https://github.com/jellyfin/Swiftfin/pull/980#discussion_r1614395534

I'm having a bit of trouble getting this to work. I have a version that works where I move the .ignoresSafeArea() to each of the views that would be contained in the contentLetterBarView(). I don't think that is what we want to do.

What would be my best approach to make this work in a way that doesn't get locked like this when there are more than 27 characters:

<details>
  <summary>Korean Letter Picker</summary>
![Screenshot 2024-05-25 at 1 01 16 PM](https://github.com/jellyfin/Swiftfin/assets/37599297/5beefc5e-89b5-40c1-9079-c057db89b873)
</details>